home.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
```

Comment:
```suggestion
    <meta charset="utf-8">
```

We're in the world of HTML5.

Reply:
Sorry for that it came out of nowhere